SofaScoreAcademy/SofaScoreAcademy/Models/Event.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  Event.swift
+//  SofaScoreAcademy
+//
+//  Created by Dominik Prce on 16.03.2024..
+//
+
+import Foundation
+
+struct League{
+    var country: String
+    var leagueName: String
+}
```

Comment:
Mozes li svuda primijeniti stil da imamo razmak nakon imena i prazan prvi red
```suggestion
struct League {

    var country: String
    var leagueName: String
}
```